nautobot/extras/datasources/git.py

```diff
@@ -631,8 +632,12 @@ def update_git_export_templates(repository_record, job_result):
                 template_record.template_code = template_content
                 modified = True
 
-            if template_record.mime_type != "text/plain":
-                template_record.mime_type = "text/plain"
+            # mimetypes.guess_type returns a tuple (type, encoding)
+            mime_type = mimetypes.guess_type(file_path)[0]
```

Comment:
Possibly `guess_type(file_path, strict=False)`? (Just going by https://docs.python.org/3/library/mimetypes.html)

Reply:
I saw that, but was easier just going with the IANA standards. I will change and push this evening.

Reply:
I don't have a strong opinion, nor is it obvious to me exactly what additional mime types `strict=False` will enable. :-) I was just educating myself about this library and saw that option, and was wondering. :-)

Reply:
In that case, I think keeping the default of `strict=True` is fine for our uses.